src/components/NotFound/Notfound.test.tsx

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 
 import NotFound from './NotFound';
 
-console.error = jest.fn();
-
 describe('<NotFound /> component', () => {
   test('should load the component in default state', () => {
```

Comment:
Perhaps one `test.todo()` for the onclick, so a next contributor can take care of it :-)